server/api.js

```diff
@@ -1,138 +1,115 @@
 import { Router } from "express";
-import db from "./db";
+import db, { pool } from "./db";
 
 const router = Router();
 
 router.get("/", (_, res) => {
 	res.status(200).json({ message: "WELCOME TO LOVE ME TENDER SITE" });
 });
 
-router.get("/skills", (req, res) => {
-	const skills = [
-		"Website",
-		"Android",
-		"iOS",
-		"Backend",
-		"Frontend",
-		"Full-stack",
-	];
-
-	skills.sort();
-
-	res.status(200).json({ skills });
-});
-
-router.post("/publish-tenders", (req, res) => {
-	const formData = req.body;
-
-	const newErrors = [];
-
-	if (
-		!formData.title ||
-		formData.title.length < 10 ||
-		formData.title.length > 50
-	) {
-		newErrors.push("Tender Title must be between 10 and 50 characters.");
-	}
-
-	if (
-		!formData.description ||
-		formData.description.length < 100 ||
-		formData.description.length > 7500
-	) {
-		newErrors.push(
-			"Tender Description must be between 100 and 7500 characters."
+router.get("/skills", async (req, res) => {
+	try {
+		const result = await db.query(
+			"SELECT skill_id, skill_name FROM skill ORDER BY skill_name ASC"
 		);
+		res.status(200).json({ skills: result.rows });
+	} catch (error) {
+		res.status(500).json({
+			code: "SERVER_ERROR",
+			error: "Failed to fetch skills. Please try again later.",
```

Comment:
This message is unnecessary. The `500` is enough to tell the client that we failed